src/storage/src/render/persist_sink.rs

```diff
@@ -93,6 +93,13 @@ where
         current_upper.borrow_mut().clear();
     }
 
+    // All sub-sources increment the same stats object.
```

Comment:
This commend is incorrect now, right? Because each sub source has their own stats object.